docs/buildah-rmi.md

```diff
@@ -1,19 +1,19 @@
-## buildah-rmi "March 2017"
+## buildah-rmi "1" "March 2017" "buildah"
 
 ## NAME
-buildah rmi - Removes one or more images. 
-
+buildah rmi - Removes one or more images.
 
 ## SYNOPSIS
-**buildah** **rmi** **imageID(s)** 
+**buildah** **rmi** **imageID [...]**
 
 ## DESCRIPTION
-Removes a locally stored image or images.  Multiple images are space separated.  If multiple images are passed to this command and the removal fails on one, the images following that image will not be removed.   
+Removes one or more locally stored images.
```

Comment:
Your call, but you're thinking we don't need the heads up on what happens if the removal of an image in the middle of the list isn't warranted?